### ISSUE TITLE
crono log take full disk space

### ISSUE THREAD

**Issue by depay** [SUSE/Portus#505]
log/development.log take full space of my disk after some days running with docker-compose...and i finally found two things:

1. docker/crono-supervisord.conf:`command=/bin/bash -c "CATALOG_CRON=10.seconds bundle exec crono"`. i'm wondering whether 10s is reasonable..

2. every line in log is too long, even beyond a whole screen. And seems `\nl` does not work.

~~~
... [2015-10-28T02:16:03.744327 #44]  INFO -- : Finished CatalogJob in 0.22 seconds\nI, [2015-10-28T02:16:13.528509 #44]  INFO -- : Perform CatalogJob\nI, [2015-10-28T02:16:13.741272 #44]  INFO -- : Finished CatalogJob in 0.21 seconds\nI, [2015-10-28T02:16:23.528857 #44]  INFO -- : Perform CatalogJob\nI, [2015-10-28T02:16:23.748565 #44]  INFO -- : Finished CatalogJob in 0.22 seconds\nI, [2015-10-28T02:16:33.529121 #44]  INFO -- : Perform CatalogJob\nI, [2015-10-28T02:16:33.756597 #44]  INFO -- : Finished CatalogJob in 0.23 seconds\n', `updated_at` = '2015-10-28 02:16:33' WHERE `crono_jobs`.`id` = 1...
~~~

**Comment by mssola** [comment]
> log/development.log take full space of my disk after some days running with docker-compose...

So, is it development.log or crono.log which is taking the full disk space ?

> 1. docker/crono-supervisord.conf:command=/bin/bash -c "CATALOG_CRON=10.seconds bundle exec crono". i'm wondering whether 10s is reasonable..

Fixed with commit: e13e9bc0c2319015cf191c943e0f9a87dcdb4e6e.

> 2. every line in log is too long, even beyond a whole screen. And seems \nl does not work.

I've noticed this. Let me take a look into that.

**Comment by depay** [comment]
@mssola ,it's development.log and thanks for help

**Comment by GrantStreetGroup** [comment]
This also effect production.log.. and the crono_jobs table.    It seems that each time it runs, it records all output from all previous runs as the log entry in the table.  Then that SQL statement is recorded by rails into the $ENV$.log

--
Steven

**Comment by mssola** [comment]
@GrantStreetGroup that's weird. We fixed the log level in production the commit f72fb28670df6a14a2d87a8cd5e1ca601797e7b8. After this, SQL statements should not be logged, for example. Maybe you're running an older version ? Maybe you updated the code but forgot to restart the server ?

**Comment by JonathonReinhart** [comment]
I'm running v1.1 from RPM and seeing this in `production.log`:

```
E, [2015-12-09T16:25:56.423804 #1093] ERROR -- : Mysql2::Error: Data too long for column 'log' at row 1: UPDATE `crono_jobs` SET `last_performed_at` = '2015-12-09 21:25:55', `log` = 'I, [2015-10-27T18:10:46.339608 #4000]  INFO -- : Perform CatalogJob\nI, [2015-10-27T18:10:46.355322 #4000]  INFO -- : Finished CatalogJob in 0.02 seconds\nI, [2015-10-27T18:20:46.340774 #4000]  INFO -- : Perform CatalogJob\nI, [2015-10-27T18:20:46.349374 #4000]  INFO -- : Finished CatalogJob in 0.01 seconds\nI, [2015-10-27T18:30:46.341795 #4000]  INFO -- : Perform CatalogJob\nI, [2015-10-27T18:30:46.349152 #4000]  INFO -- : Finished CatalogJob in 0.01 seconds\nI, [2015-10-27T18:40:46.342652 #4000]  INFO -- : Perform CatalogJob\nI, [2015-10-27T18:40:46.350150 #4000]  INFO -- : Finished CatalogJob in 0.01 seconds\nI, [2015-10-27T18:50:46.343675 #4000]  INFO -- : Perform CatalogJob\nI, [2015-10-27T18:50:46.356135 #4000]  INFO -- : Finished CatalogJob in 0.01 seconds\nI, [2015-10-27T19:00:46.344519 #4000]........
```
